mfr/extensions/utils.py

```diff
@@ -0,0 +1,22 @@
+from urllib.parse import urlparse, parse_qs, urlencode
+from mfr.extensions import settings
+
+
+def download_from_template(func):
```

Comment:
The decorator looks good to me, but I'd like to bikeshed the name, `download_from_template` is a little nonspecific.  Maybe `munge_url_for_localdev` or something like that?  Or even `munge_download_url` if this might end up doing more than just rewriting the localdev download url.  What do you think?